CorePayments/src/main/java/com/paypal/android/corepayments/api/GetClientTokenResult.kt

```diff
@@ -0,0 +1,21 @@
+package com.paypal.android.corepayments.api
+
+import androidx.annotation.RestrictTo
+import com.paypal.android.corepayments.PayPalSDKError
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+sealed class GetClientTokenResult {
```

Comment:
```suggestion
sealed class CreateLowScopedAccessTokenResult {
```

🧶 : to align result name with function method name.